ansible/galaxy.yml

```diff
@@ -0,0 +1,46 @@
+namespace: marinatedconcrete
+name: config
+
+# The version of the collection. Must be compatible with semantic versioning
+version: 1.0.0
+
+readme: README.md
+
+# A list of the collection's content authors. Can be just the name or in the format 'Full Name <email> (url)
+# @nicks:irc/im.site#channel'
+authors:
+- Marinated Concrete Authors
```

Comment:
Maybe just put us both here?

Reply:
Sure thing, was just lazy and didn't want to figure out what we actually required.